cjwstate/modules/types.py

```diff
@@ -274,3 +282,17 @@ def get_param_schema_version(self) -> str:
             # "develop" is handled specially in cjwstate/params.py. For GitHub
             # sha1, a new version forces migrate_params().
             return self.version
+
+    @lru_cache(maxsize=len(supported_locales))
```

Comment:
Please nix this and rename `_read_bytes()` to `read_bytes()`.

Reply:
Nix caching of the whole `get_message_catalog` method? I think it's better to keep `get_message_catalog`, in order to define the catalog path in there.

Reply:
Yes, nix caching.

You're right that it's a good idea to define the path. Since it won't return a catalog, shall we call it ... uh ... `read_messages_po_for_locale()`? (That name doesn't roll off the tongue, but it's important that we describe the bytestream format, because the return type will simply be `bytes`.)